documentation/docs/release-notes/3.0.0_1.md

```diff
@@ -0,0 +1,116 @@
+# Percona Monitoring and Management 3.0.0-1
+
+**Release date**: February 10th, 2025                                                                     
+
+
+Percona Monitoring and Management (PMM) is an open-source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB. 
+
+It provides tools to observe database health, analyze performance trends, troubleshoot issues, and execute database management tasks—whether your databases are on-premises or in the cloud.
+
+
+## SEVERITY: CRITICAL - IMMEDIATE ACTION REQUIRED
+
+A critical vulnerability has been identified in PMM Open Virtual Appliance (OVA) installations that enables unauthorized `root` access and potential exposure of system credentials. This vulnerability is tracked as [CVE-xxxx-xxxx](https://cve.mitre.org/cgi-bin/cvename.cgi?name=xxxx-xxxx).
+
+### Immediate actions required
+
+1. UPGRADE IMMEDIATELY to PMM 3.0.0-1 (strongly recommended).
+2. CHANGE ALL CREDENTIALS for connected services and databases.
+3. AUDIT ACCESS LOGS for potential unauthorized access.
+
+### Vulnerability details
+This vulnerability stems from default service account credentials in OVA provisioning that enables:
+
+ - unauthorized SSH access
+ - privilege escalation to `root` via `sudo` capabilities
+ - potential exposure of service credentials and configurations
+
+### Affected installations  
+The table below lists currently known affected deployments. We will update it if additional products are identified:
+
+| Affected deployments                     | Version | Notes |
+|-------------------------------------------|---------|-------|
+| PMM Open Virtual Appliance (OVA) installations | ≥ 2.38  |       |
+
+### Mitigation options  
+
+
+=== "PREFERRED: Upgrade to PMM 3.0.0-1"
+    This release enhances security in OVA deployments by automatically removing unnecessary system accounts during the initial setup.
+
+    To secure your system, follow these steps to upgrade:
+
+    1. Download and deploy the new OVA file from [Percona website](https://www.percona.com/downloads).
+
+    2. Stop services on your current installation: 
+        ```sh 
+        supervisorctl stop all
+        ```
+    3. Back up and transfer your data:
+        ```sh
+        cd /srv
```

Comment:
`/home/admin/volume/srv`

Reply:
the comment was to replace /srv with `/home/admin/volume/srv`, not the whole code block. here and below